mne/externals/FieldTrip.py

```diff
@@ -309,7 +309,7 @@ def getHeader(self):
                 (chunk_type, chunk_len) = struct.unpack(
                     'II', payload[offset:offset + 8])
                 offset += 8
-                if offset + chunk_len < bufsize:
+                if offset + chunk_len > bufsize:

```

Comment:
@jasmainak without the change, I get weird failures streaming directly from the TRIUX machine. It never works on the first try, but after stopping and starting the recording, it starts to work. (FieldTrip Matlab client works fine all the time)

I think it's a subtle bug. The code is intended to check whether the next chunk would overshoot  the byte array (`payload`) and bail out in that case, but instead it bails out whenever `offset + chunk_len` is is shorter than the payload. However in this way it ends up working on any occasion where `offset + chunklen` exactly equals the length of the payload, which often seems to be the case (meaning, I guess, that there is only one chunk to read).